src/chronify/models.py

```diff
@@ -74,6 +74,35 @@ def list_columns(self) -> list[str]:
         return super().list_columns() + [self.value_column]
 
 
+class PivotedTableSchema(TableSchemaBase):
```

Comment:
This is similar to what may exist in a CSV file. Instead, the user has the data in memory.